_topic_maps/_topic_map.yml

```diff
@@ -592,7 +592,9 @@ Topics:
 - Name: Updating a cluster using the web console
   File: updating-cluster-within-minor
 - Name: Updating a cluster using the CLI
-  File: updating-cluster-cli
+  File: updating-cluster-cli 
+- Name: Migrating clusters to a multi-arch/heterogeonous payload
```

Comment:
Is "multi-arch/heterogeneous" an official glossary term? :D It feels like we should select one of these redundant terms and go with it...